### PR TITLE
Contributing guide and Code of conduct

### PR DESCRIPTION
**Pull request by fcor**
Hi!

Just wrapping up a small contribution guide and a code of conduct for the community.

Please take a look and let me know what do you think 😃

### CODE REVIEW

**Comment by fcor** [comment]
I think we can also update some things on readme, like adding new links and this files for example

**Comment by nicolocarpignoli** [comment]
Great Fabio! I just added trivial notes and typos 👍

**Comment by fcor** [comment]
@nicolocarpignoli I can't see the notes. Thanks for reading and helping! 👍

**Comment by nicolocarpignoli** [comment]
Now you should be able to!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+5. If possible, provide a a link to your application, and/or a screenshot.
+
+# Contribute Code to AR.js Studio
+
+[studio]:https://github.com/AR-js-org/studio
+[first]: https://github.com/AR-js-org/studio/issues?q=is%3Aissue+is%3Aopen+label%3A1.0.0
+[hacktoberfest]: https://github.com/AR-js-org/studio/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest
+[pr]: https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github
+[ssh]: https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
+[testing-guide]: https://github.com/aframevr/aframe/tree/master/tests#a-frame-unit-tests
```
[the remark this answers is not in the document]
Comment (fcor) [reply CONTRIBUTING.md]: You mean the whole section? or just the guides (PR, SSH etc)?

**Review by gciandro13 — COMMENTED** [review]
Great for me!! Thank you for your job fabio

**Pull request merged**